Document/0x06b-Basic-Security-Testing.md

```diff
@@ -167,13 +166,10 @@ After adding all the suggested repositories above you can install the following
 - Cycript: Is an inlining, optimizing, Cycript-to-JavaScript compiler and immediate-mode console environment that can be injected into running processes (associated to Substrate).
 - Cydia Substrate: A platform that makes developing third-party iOS add-ons easier via dynamic app manipulation or introspection.
 - cURL: Is a well known http client which you can use to download packages faster to your device. This can be a great help when you need to install different versions of Frida-server on your device for instance.
-- Darwin CC Tools: Install the Darwin CC Tools from the Coolstar repo as a dependency for Needle.
```

Comment:
Let's leave this in, but change the description:

- Darwin CC Tools: A useful set of tools like nm, and strip that are capable of auditing mach-o files.